src/Polly.Core/Utils/StrategyHelper.cs

```diff
@@ -4,6 +4,10 @@
 
 internal static class StrategyHelper
 {
+    [DebuggerDisableUserUnhandledExceptions]
+#if !DEBUG
+    [DebuggerNonUserCode]
```

Comment:
You don't need `[DebuggerNonUserCode]`. Sorry, I should have added a comment to my sample -- that was only needed in my sample because I was adding code to a single project and I wanted the debugger to treat some of it as if it was library code.